Jenkinsfile

```diff
@@ -259,9 +276,13 @@ pipeline {
 							dir("libs/${PLATFORM}") {
 								unstash "swt.binaries.${PLATFORM}"
 								sh '''
+									DEST_PLATFORM=${PLATFORM}
 									if [[ ${PLATFORM} == cocoa.macosx.* ]]; then
 										binariesExtension='jnilib'
 										signerUrl='https://cbi.eclipse.org/macos/codesign/sign'
+									elif [[ ${PLATFORM} == gtk4.linux.x86_64 ]]; then
+										binariesExtension='so'
+										DEST_PLATFORM=gtk.linux.x86_64
```

Comment:
We could probably save this block when using a slightly adjusted pattern in the next block(given the change below is applied):
`elif [[ ${PLATFORM} == gtk*.linux.* ]]; then`

But it's probably simpler to understand if we are explicit there. On the other hand we could still be explicit and save this block if we just combine both block and 'or' the arguments.

Reply:
See the updated commit when I submit, but I reworked this section to make it cleaner and achieved something I wanted to do originally.